src/js/menu/menu-button.js

```diff
@@ -8,6 +8,7 @@ import * as Dom from '../utils/dom.js';
 import * as Fn from '../utils/fn.js';
 import * as Events from '../utils/events.js';
 import toTitleCase from '../utils/to-title-case.js';
+import IS_IOS from '../utils/browser.js';
```

Comment:
`IS_IOS` needs to be wrapped in curly braces when you want a named import
```js
import { IS_IOS } from '../utils/browser.js';
```

Reply:
This is also why the build is failing.

Reply:
_facepalm_. Fixed.